pkg/runtime/runtime_test.go

```diff
@@ -1577,6 +1570,13 @@ func TestOnNewPublishedMessageGRPC(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
+			port, _ := freeport.GetFreePort()
+			rt := NewTestDaprRuntimeWithProtocol(modes.StandaloneMode, string(GRPCProtocol), port)
+			rt.topicRoutes = map[string]TopicRoute{}
+			rt.topicRoutes[TestPubsubName] = TopicRoute{routes: make(map[string]string)}
+			rt.topicRoutes[TestPubsubName].routes["topic1"] = "topic1"
```

Comment:
```suggestion
			rt.topicRoutes[TestPubsubName] = TopicRoute{
				routes: map[string]string{
					topic: topic,
				},
			}
```

Reply:
You could also do the same with `topicRoutes` but I guess that's one more nesting.